examples/pytorch/speech-recognition/run_speech_recognition_ctc.py

```diff
@@ -599,6 +615,27 @@ def remove_special_characters(batch):
     # derive max & min input length for sample rate & max duration
     max_input_length = data_args.max_duration_in_seconds * feature_extractor.sampling_rate
     min_input_length = data_args.min_duration_in_seconds * feature_extractor.sampling_rate
+
+    if data_args.max_eval_duration_in_seconds is None:
+        logger.warning(
+            "`max_eval_duration_in_seconds` is not provided. Filtering audio samples in the eval set longer than"
+            " `max_duration_in_seconds`. From Transformers v5, the eval set will not be filtered by maximum audio"
+            " length if `max_eval_duration_in_seconds` is not provided."
+        )
+        max_eval_input_length = max_input_length
+    else:
+        max_eval_input_length = data_args.max_eval_duration_in_seconds * feature_extractor.sampling_rate
+
+    if data_args.min_eval_duration_in_seconds is None:
+        logger.warning(
```

Comment:
Same comments as above - just no need to allow a boolean. But we should incentivize the reader to set `min_eval_duration_in_seconds` to 0